lib/services/local/access_list.go

```diff
@@ -303,6 +321,30 @@ func (a *AccessListService) GetAccessListMember(ctx context.Context, accessList
 	return member, trace.Wrap(err)
 }
 
+// GetAccessListOwners returns a list of all owners in an Access List, including those inherited from nested Access Lists.
+//
+// Returned Owners are not validated for ownership requirements – use `IsAccessListOwner` for validation.
+func (a *AccessListService) GetAccessListOwners(ctx context.Context, accessListName string) ([]*accesslist.Owner, error) {
```

Comment:
Nit: There's a few potential sources of errors w/in this function. Introducing error wrapping will help us pin-point the source in future, e.g:

```
	accessLists, err := a.GetAccessLists(ctx)
	if err != nil {
		return nil, trace.Wrap(err, "getting access lists")
	}
	hierarchy, err := accesslists.NewHierarchy(ctx, accesslists.HierarchyConfig{
		AccessLists: accessLists,
		Members:     a,
		Locks:       nil,
		Clock:       a.clock,
	})
	if err != nil {
		return nil, trace.Wrap(err, "creating hierarchy")
	}
	owners, err := hierarchy.GetOwners(accessListName)
	if err != nil {
		return nil, trace.Wrap(err, "getting owners")
	}
```